tests/unit_test/apis/dxo_test.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
since we publish this code 2023 should change it to 2023?

Otherwise all logic LGTM.

Reply:
Right. Updated to 2023.